### PR TITLE
IS-2231 Slack integration with notifications

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/getting-started/postman-account.md [hunk 1]
```diff
@@ -180,10 +180,12 @@ You can enable or disable your public profile at any time in your [profile setti
 
 You can update your [notification preferences](https://go.postman.co/settings/me/notifications) by selecting your avatar in the upper-right corner > **Notification Preferences**.
 
-You can opt in to or out of email or in app notifications about security, usage, monitors, and comments by selecting or de-selecting the boxes next to each item. Select **Update Preferences** to save changes.
+You can opt in to or out of email or in-app notifications about security, usage, monitors, and comments by selecting or de-selecting the boxes next to each item. Select **Update Preferences** to save changes.
 
 <img alt="Update notification preferences" src="https://assets.postman.com/postman-docs/notification-preferences-v9.jpg">
```
Comment (shashankawasthi88) [inline src/pages/docs/getting-started/postman-account.md:185]: @jkonrath-postman  Can we add new screenshot with the Slack Notification preference as well

**Review by shashankawasthi88 — COMMENTED** [review]
Looks good, just need to update the Notification center image.

File: src/pages/docs/getting-started/postman-account.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/getting-started/postman-account.md]: Added this image here, and in the slack.md itself.

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 2]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
 ## Configuring Postman with Slack
 
-From the **[Home](https://go.postman.co/home)** page select **[Integrations](https://go.postman.co/integrations)**.
+1. From the **[Home](https://go.postman.co/home)** page select **[Integrations](https://go.postman.co/integrations)**.
+1. Select **Browse All Integrations**.
+1. Search for and select **Slack**.
+1. On the [Slack integration details](https://go.postman.co/integrations/service/slack), there are three available integrations:
 
-![home page and integrations](https://assets.postman.com/postman-docs/home-integrations.jpg)
+    * **[Post monitoring results](#send_your_monitor_run_results_to_slack)** - Send the results from a monitor run to a specified Slack channel.
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/slack.md:36]: Link not working, use hyphens not underscores

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 3]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
 
-Search and select **Slack**.
+    * **[Receive Postman Notification](#receive-postman-notifications-in-slack)** - Send notifications received in the Postman notification center to a specified Slack channel.
 
-![search for slack integration](https://assets.postman.com/postman-docs/slack-search-all.jpg)
+    * **[Post team activity](#add_an_activity_feed_to_slack)** - Send your team's [activity feed](/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections/#accessing-the-activity-feed-from-postman) to a specified Slack channel.
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/slack.md:40]: Link not working, use hyphens not underscores

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 4]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
 
-From the integration details page you can choose to:
+        ![select Slack integration](https://assets.postman.com/postman-docs/slack-select-integration.jpg)
 
-* Post team activity
+1. A browser window to log in to Slack opens. If you are not signed in to a Slack workplace, enter your workspace URL and sign in with your email and password or SSO.
+    > Note: this window may be blocked by your pop-up blocker.
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/available-integrations/slack.md:45]: You can delete "Note:" - blockquote style already indicates it's a note

**Review by mark-anthony-postman — APPROVED** [review]
A few minor comments, otherwise LGTM!

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 5]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
-![configured_slack](https://assets.postman.com/postman-docs/slack-post-monitoring-results-authorized.jpg)
+> Before beginning, you must create a monitor. For more information, see [setting up a monitor](/docs/monitoring-your-api/setting-up-monitor/).
 
-Return to the browser tab for Postman and enter the following:
+1. After completing the above steps, enter the following in the **Add integration** screen:
 
-* Nickname
+    * **Nickname** -   A nickname for your integration.
+    * **Workspace** -   The workspace containing your monitor.
+    * **Monitor** -   The monitor which will send its results to Slack.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/slack.md:67]:
```suggestion
    * **Monitor** -  The monitor that will send its results to Slack.
```

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 6]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
-![configured_slack](https://assets.postman.com/postman-docs/slack-post-monitoring-results-permission.jpg)
-
-![configured_slack](https://assets.postman.com/postman-docs/slack-post-monitoring-results-authorized.jpg)
+> Before beginning, you must create a monitor. For more information, see [setting up a monitor](/docs/monitoring-your-api/setting-up-monitor/).
 
-Return to the browser tab for Postman and enter the following:
+1. After completing the above steps, enter the following in the **Add integration** screen:
 
-* Nickname
+    * **Nickname** -   A nickname for your integration.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/slack.md:65]:
```suggestion
    * **Nickname** -  A nickname for your integration.
```

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 7]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
+    * **Workspace** -   The workspace containing your monitor.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/slack.md:66]:
```suggestion
    * **Workspace** -  The workspace that contains your monitor.
```

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 8]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
+    * **Monitor** -   The monitor which will send its results to Slack.
+    * **Slack Webhook URL** - The webhook URL which was pre-filled from the authorization process.
+    * **Slack Channel** - The channel where the integration will post, which is pre-filled from the authorization process.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/slack.md:69]:
```suggestion
    * **Slack Webhook URL** - The webhook URL. This field will be pre-filled from the authorization process.
    * **Slack Channel** - The channel where the integration will post. This field will be pre-filled from the authorization process.
```

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 9]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
-  The workspace your monitor belongs to.
-
-* Monitor
+The following is an example of a set of monitor results when sent to Slack:
 
-  The results of the monitor will be sent to Slack.
+![configured_slack_example](https://assets.postman.com/postman-docs/slack-post-monitoring-results-example-v9.jpg)
 
-* Slack Webhook URL
+## Receive Postman Notifications in Slack
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/slack.md:78]: Should this heading be sentence case? I thought I saw that rule either in the LC or Content Design style guides - can't find it now, but here's the Microsoft style guide entry: https://docs.microsoft.com/en-us/style-guide/scannable-content/headings#formatting-headings

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 10]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
+<img alt="Update notification preferences" src="https://assets.postman.com/postman-docs/notification-preferences-v9-4.jpg">
 
-    * Notify for completed monitor runs
-    * Notify for 3 failures and the first success
->
-    > You can click the Advanced Options link to indicate if you want notifications for all completed monitor runs, or notifications for three failed monitor runs and then the first successful monitor run.
+In the Slack column, you can opt in to or out of notifications such as security, usage, monitors, and comments. Select or de-select the boxes next to each item. Select **Update Preferences** to save changes.
 
-![configured_slack](https://assets.postman.com/postman-docs/slack-post-monitoring-results-save-config.jpg)
+## Add an Activity Feed to Slack
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/slack.md:88]: Same question as above about sentence case.

File: src/pages/docs/integrations/available-integrations/slack.md [hunk 11]
```diff
@@ -23,96 +24,69 @@ contextual_links:
...
-In the **Select channel** page, select the channel where you want to post and click the **Authorize** button.
-
-[![auth_slack](https://assets.postman.com/postman-docs/WS-integrations-slack-identity.png)](https://assets.postman.com/postman-docs/WS-integrations-slack-identity.png)
-
-The team activity appears in the **Configured Integrations** page.
-
-[![configured_slack](https://assets.postman.com/postman-docs/WS-integrations-slack-configured.png)](https://assets.postman.com/postman-docs/WS-integrations-slack-configured.png)
+1. Return to Postman, and complete the steps in the relevant section below for your integration type.
 
 ## Send your Monitor Run Results to Slack
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/slack.md:59]: Same question about sentence case for headings.

**Review by HeatherStenson — APPROVED** [review]
Left a few comments and questions, none of which are blockers!

**Pull request merged**